docs/developers/deploy-on-polygon.md

```diff
@@ -1,27 +1,27 @@
 ---
-title: Deploying the LSP on Polygon
-sidebar_label: Deploying the LSP on Polygon
+title: Deploying an LSP Contract
+sidebar_label: Deploying an LSP Contract
 ---
-UMA's infrastructure is also available on Polygon for developers to make use of as a scaling solution. The following will explain the details on how to deploy on Polygon. 
+This tutorial will show you how to deploy a Long Short Pair (LSP) contract by using the UMA `launch-lsp` repository. By following these instructions, you will be able to launch a custom LSP on Mumbai testnet, Polygon mainnet, Kovan testnet, and Ethereum mainnet.
 
-Deploying to Polygon follows a near-identical launch process as to launching any UMA contract on Ethereum. Using one of our launch repo's and amending the destination network, you can quickly deploy your contracts on Polygon.
+UMA's infrastructure is available on Polygon for developers to make use of as a scaling solution. Deploying to Polygon follows a near-identical launch process as to launching any UMA contract on Ethereum. Using one of our launch repo's and amending the destination network, you can quickly deploy your contracts on Polygon.
 
 ## Deployment process
 
-At the end of this deployment process, you should have a deployed contract on Polygon mainnet or Mumbai testnet. The following is an overview of the deployment steps below.
+At the end of this deployment process, you should have a deployed contract on Mumbai testnet, Polygon mainnet, Kovan testnet, or Ethereum mainnet. The following is an overview of the deployment steps below.
 
 1. Clone the repo
 2. Install the dependencies
 3. Set your parameters
-5. Run the deployment on Mumbai testnet
-6. Run the deployment on Polygon mainnet  
-7. Connect to a financial product library  
+4. Run the deployment on testnet (Mumbai or Kovan)
+5. Run the deployment on mainnet (Polygon or Ethereum mainnet) 
+6. Connect to a financial product library
 
 ### Is your desired price identifier and collateral type supported?
 
-Please check with the UMA team if your collateral and price identifiers have been approved for use on Polygon. There will be a subset of commomly used collaterals that will be available on Polygon at launch. 
+Please check with the UMA team if your collateral and price identifiers have been approved for use on the network you wish to deploy your contract to.
```

Comment:
Can we link to [approved price identifiers](https://docs.umaproject.org/uma-tokenholders/approved-price-identifiers) and [approved collateral currencies](https://docs.umaproject.org/uma-tokenholders/approved-collateral-currencies) here instead?